packages/roosterjs-plugin-picker/lib/PickerPlugin.ts

```diff
@@ -52,6 +52,10 @@ export default class EditorPickerPlugin implements EditorPickerPluginInterface {
                 this.setIsSuggesting(isSuggesting);
             }
         );
+
+        if (this.dataProvider.setEditor) {

```

Comment:
Thoughts about this being part of `onInitialize` instead of it's own separate function?

Reply:
Yeah, I was going to do that. Just means we have to do a major version bump because it would be a breaking change.

Reply:
adding to the api as an optional (non required) parameter shouldn't be, that's an additive API change. Unless you're arguing to make it required?

Reply:
Yeah, good point. I was thinking to make it required (since the plugin will always be providing it), but it's probably cleaner to include it as an optional argument instead of a different method.